templates/waggledance.json

```diff
@@ -50,7 +55,11 @@
 	  {
 		"name": "LOG4J_FORMAT_MSG_NO_LOOKUPS",
 		"value": "true"
-	  }
+	  },
+	{
+		"name": "DD_USE_DOGSTATSD",
```

Comment:
does this need to be parameterised here in the module  ?

Reply:
no this is set to true by default. but will be used only when datadog-agent is running.

Reply:
what is the use of this variable ? what happens if agent is running and this is set to false?

Reply:
Removed it as the default value is true.